apps/L1G/tinder/tinder-frontend/src/components/ChatWindow.tsx

```
@@ -0,0 +1,77 @@
+            >
+              Unmatch
+            </Button>
+          </div>
+        </div>
+      </div>
+
+      <div className="flex flex-col justify-center items-center flex-grow gap-2 mt-8">
+        <MessageSquareDashedIcon size={48} color="gray" />
+        <p className="text-[14px] font-medium">Say Hi!</p>
+        <p className="text-[14px] text-[#71717A] text-center max-w-[358px]">
+          You've got a match! Send a message to start chatting.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)